wren-ai-service/src/pipelines/ask/indexing_pipeline.py

```diff
@@ -23,46 +23,112 @@
 DATASET_NAME = os.getenv("DATASET_NAME")
 
 
-class Indexing(BasicPipeline):
+@component
+class DocumentCleaner:
+    """
+    This component is used to clear all the documents in the specified document store(si).
+
+    """
+
+    def __init__(self, stores: List[DocumentStore]) -> None:
+        self._stores = stores
+        pass
+
+    @component.output_types(mdl=str)
+    def run(self, mdl: str) -> str:
+        def _clear_documents(store: DocumentStore) -> None:
+            ids = [str(i) for i in range(store.count_documents())]
+            if ids:
+                store.delete_documents(ids)
+
+        logger.info("Ask Indexing pipeline is clearing old documents...")
+        [_clear_documents(store) for store in self._stores]
```

Comment:
I think we should not directly delete all document in all stores here.

Reply:
after discussion, there is no issue deleting all documents in all stores. since we assume mdl should contain the latest spec.

Reply:
after the discussion, we won't modfiy the behavior here.